packages/dashboard/src/components/elements/frame/frame.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+const Frame: React.FC<TFrameProps> = ({ src, alt, className }) => {
+    const image_ref = React.createRef<HTMLImageElement>();
```

Comment:
Can't use `useRef` here?
I think it's more suitable for function components since `createRef` always returns a new ref which is useful in class components.

Reply:
You're right, it does create a new ref each time which is definitely not ideal. However this is a temporary page so let's not worry about this logic too much, this page will be removed quite soon.